pages/common/gource.md

```diff
@@ -0,0 +1,31 @@
+# gource
+
+> Visualize the history of Git, SVN, Mercurial and Bazaar repositories.
+
+- Run gource in a directory:
```

Comment:
Already made some changes earlier (because of other inline comments). The description got kinda long so I'm not sure how to reword it.